tests/integ/test_tf.py

```diff
@@ -22,6 +22,8 @@
 from tests.integ.timeout import timeout_and_delete_endpoint_by_name, timeout
 
 DATA_PATH = os.path.join(DATA_DIR, 'iris', 'data')
+VPC_SUBNETS = ['subnet-06b8537735fac3757']
```

Comment:
Is this dependent on our account?

Reply:
Yep. I created this VPC in PDX. The test I added the vpc_config is not used in canary so we are not running it in other regions.

Reply:
Can we create the VPC configurations and subnet during the test? Are we guaranteed to run this VPC config in the same account all the time?

I think the test will be better suited if we make these changes.

Reply:
We certainly can create the VPC in the test and delete it when the test finishes. I understand your concern. On the other hand I do not want our test to depend on the availability of vpc. Perhaps we can pre create vpc configs for all the regions? I am not too keen on this solution either since we might want to run the tests in different accounts in the future. Thoughts?

Reply:
Create if not exists?  This should be a good tradeoff